src/rsem/rsem_prepare_reference/config.vsh.yaml

```diff
@@ -0,0 +1,193 @@
+name: rsem_prepare_reference
+namespace: rsem
+description: | 
+  RSEM is a software package for estimating gene and isoform expression levels from RNA-Seq data. This component prepares transcript references for RSEM.
+keywords: ["Transcriptome", "Index"]
+links:
+  homepage: https://github.com/deweylab/RSEM
+  documentation: https://deweylab.github.io/RSEM/rsem-prepare-reference.html
+  repository: https://github.com/deweylab/RSEM?
+references: 
+  doi: 10.1186/1471-2105-12-323
+license: GPL-3.0 
+requirements:
+  commands: [ rsem-prepare-reference ]
+
+argument_groups:
+  - name: Inputs
+    arguments:
+      - name: --reference_fasta_files
+        type: file
+        description: | 
+          List of Multi-FASTA formatted files OR a directory name. If a directory name is specified, RSEM will read all files with suffix ".fa" or ".fasta" in this directory. The files should contain either the sequences of transcripts or an entire genome, depending on whether the '--gtf' option is used.
+        required: true
+        example: read1.fasta
+      - name: --reference_name
+        type: string
+        description: | 
+          The name of the reference used. RSEM will generate several reference-related files that are prefixed by this name. This name can contain path information (e.g. '/ref/mm9').
+        required: true
+        example: /ref/mm9 
+  
+  - name: Outputs
+    arguments:
+      - name: --output
+        type: file
+        description: Directory containing reference files generated by RSEM. 
+        required: true
+        direction: output
+
+  - name: Other options
+    arguments: 
+      - name: --gtf
+        type: file
+        description: Assume that 'reference_fasta_files' contains the sequence of a genome, and extract transcript reference sequences using the gene annotations specified in the GTF file. If this and '--gff3' options are not provided, RSEM will assume 'reference_fasta_files' contains the reference transcripts. In this case, RSEM assumes that name of each sequence in the Multi-FASTA files is its transcript_id.
+        example: annotations.gtf
+      - name: --gff3
+        type: file
+        description: GFF3 annotation file. Converted t GTF format with the file name 'reference_name.gtf'. Please make sure that 'reference_name.gtf' does not exist. 
+        example: annotations.gff
+      - name: --gff3_rna_patterns
+        type: string
+        description: Comma-separated list of transcript categories, e.g. "mRNA,rRNA". Only transcripts that match the string will be extracted.
+        example: mRNA,rRNA
```

Comment:
Should this be made multiple: true?

Reply:
RSEM also expects a string of comma separated categories so I decided to use the same. In that case it shouldn't be multiple true.

Reply:
I'd still like `multiple` to be set to true. If you replace `;` with `,`, then it should work regardless of whether somebody provides multiple inputs (`--gff3_rna_patterns mRNA --gff3_rna_patterns rRNA`) or one input using the separator of the original tool (`--gff3_rna_patterns mRNA,rRNA`)